bindata/v4.0.0/controller/signing-cabundle.yaml

```diff
@@ -5,5 +5,6 @@ metadata:
   name: signing-cabundle
   annotations:
     openshift.io/owning-component: service-ca
+    openshift.io/description: "Service CA signing CA bundle"
```

Comment:
This should be more descriptive. What is it used by, for what purpose.

Reply:
Fixed.